liri.js

```diff
@@ -0,0 +1,195 @@
+	});
+}
+
+function movieThis(movie) {
+	var accessKey = keys.oMDbAccessKey.accessKey;
+	var queryURL = "http://www.omdbapi.com/?t=" + movie + "&y=&plot=short&apikey=" + accessKey;
+	request(queryURL, function(err, response, body){
+		if (err) {
+			console.log("OMDB Error: " + err);
+			logOutput("OMDB Error: " + err+'\n');
+			return;
+		}
+		if (response.statusCode != 200) {
+			console.log("Failed - OMDb Response Code: " + response.statusCode);
+			logOutput("Failed - OMDb Response Code: " + response.statusCode+'\n');
+			return;
+		}
+
+		var logStr = ""; // Save the output to this string and display it on console and append to log.txt
+		var result = JSON.parse(body);
+		logStr = logStr+"Title: " + result.Title+'\n';
+		logStr = logStr+"Year: " + result.Year+'\n';
+
+		if (result.Ratings.length == 0) {
+			logStr = logStr+"Internet Movie Database Rating: Not Available"+'\n';
+			logStr = logStr+"Rotten Tomatoes Rating: Not Available"+'\n';
+		}
+
+		if (result.Ratings.length == 1) {
+			logStr = logStr+"Internet Movie Database Rating: " + result.Ratings[0].Value+'\n';
+			logStr = logStr+"Rotten Tomatoes Rating: Not Available"+'\n';	
+		}
+
+		if (result.Ratings.length == 2) {
+			logStr = logStr+"Internet Movie Database Rating: " + result.Ratings[0].Value+'\n';
+			logStr = logStr+"Rotten Tomatoes Rating: " + result.Ratings[1].Value+'\n';
+		}
+		
+		logStr = logStr+"Country: " + result.Country+'\n';
+		logStr = logStr+"Language: " + result.Language+'\n';
+		logStr = logStr+"Plot: " + result.Plot+'\n';
+		logStr = logStr+"Actors: " + result.Actors+'\n';
+		console.log(logStr);
+		logOutput(logStr);
+	});
+}
+
+function doWhatItSays() {
+	fs.readFile("random.txt", "utf8", function(err, data){
+		if (err) {
+			console.log("Error: " + err);
+			return;
+		}
+		var strArray = data.split(',');
+		var commandStr  = strArray[0];
+		var str = strArray[1];
+
+		runCommand(commandStr, str);
+	});
+}
+
+function runCommand(command, str) {
+	switch (command) {
+	 	case 'my-tweets':
+	 		var tweet = str;
+	 		if (!tweet) {tweet = 'BarackObama';}
+	 		myTweets(tweet);
+	 		break;
+		case 'spotify-this-song':
+			var song = str;
+			if (!song) {song = 'The Sign';}
+			spotifyThisSong(song);
+			break;
+		case 'movie-this':
+			var movie = str;
+			if (!movie) {movie = 'Mr. NoBody';}
+			movieThis(movie);
+			break;
+		case 'do-what-it-says':
+			doWhatItSays();
+			break;
+		case 'help':
+		default:
+			console.log();
+			console.log("node liri.js <my-tweets> | <spotify-this-song> | <movie-this> | <do-what-it-says> | <help> [--log]");
+			console.log();
+			console.log("--> my-tweets: display tweets");
+			console.log("--> spotify-this-song: get the song information from Spotify");
+			console.log("--> movie-this: get movie information from OMDb");
+			console.log("--> do-what-it-says: get random text and do what it says");
+			console.log("--> help: display help information");
+			console.log("--> --log: log output to log.txt file")
+			console.log();
```

Comment:
one console.log is enough here, it is an expensive process that doesn't need to be repeated.